src/Level.h

```diff
@@ -0,0 +1,36 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <filesystem>
+#include <string>
+
+namespace openblack
+{
+class Level
+{
+public:
+	Level(const std::string name, const std::filesystem::path path, bool isCampaign)
```

Comment:
Copies don't need to be const.
You should also `std::move` them
Unless they are meant to be references